Add a note about mocha.parallel option (#2471)
NomicFoundation/hardhat: docs/guides/parallel-tests.md

$ npx hardhat test --parallel
```

> or set `parallel: true` to the `mocha` section of the `hardhat` config file, that'll make parallelism to work for plugins like `solidity-coverage` that don't have such flag.

Reviewer: ```suggestion
Alternatively, use `parallel: true` in the `mocha` section of `hardhat.config`, which will enable parallelism for plugins that don't have a `--parallel` flag, like `solidity-coverage`.
```
